src/notification.js

```diff
@@ -294,14 +294,11 @@ export class NotificationElement extends LitElement {
       <div>
         ${iconFlask.node[0]}
         <div class="title">
-          This is the <span>latest development version</span>
-          ${this.renderCloseButton()}
+          This is the latest development version ${this.renderCloseButton()}
         </div>
         <div class="content">
-          Some features may not yet be available in the published stable
-          version. Read the
-          <a href="${this.urls.stable}">stable version of this documentation</a
-          >.
+          Some features may only be avilable in the
+          <a href="${this.urls.stable}">stable version</a>.
```

Comment:
```suggestion
          Some features may not yet be available in the
          <a href="${this.urls.stable}">stable version</a>.
```

It seems you changed the meaning of the phrase.